### PR TITLE
Updated Some Farm-related Stuff

### PR DESCRIPTION
**Pull request by EatPant2nd**
## Description

Updated Colbur's and Petaya's descriptions to include their effects: Kasib mutation prevention and the timer pause, respectively. 
Added more information to the Mulches part of the Farm's Help tab.

## Motivation and Context

Colbur and Petaya were a bit vague with their descriptions, especially Colbur where the player had no way of knowing it prevented Kasib from mutating. As for the mulches... Many times it's asked how long they last so I added that missing information to the Farm's Help tab.

## How Has This Been Tested?

Checked the respective modals.

## Types of changes

- Text update

### CODE REVIEW

File: src/scripts/farming/Farming.ts [hunk 1]
```diff
@@ -1178,7 +1179,7 @@ class Farming implements Feature {
             BerryFirmness.Very_Hard,
             [
                 'This Berry is surrounded by mystery. It is rumored to be imbued with the power of all living things.',
-                'This power keeps other Berries alive for longer.',
+                'This power resets and pauses the wither timer of other Berries indefinitely.',
```
Comment (RegisCoaxans) [inline src/scripts/farming/Farming.ts:1182]: It seems too straightforward and technical for in-game information. What do you think of this :
```suggestion
                'This power indefinitely saves other Berries from withering.',
```

Comment (EatPant2nd) [reply src/scripts/farming/Farming.ts:1182]: I mean, it's better than the original but I wanted some way to convey that it resets timers too as that was the sole reason I edited it.

Comment (CypherX) [reply src/scripts/farming/Farming.ts:1182]: > Revitalizes and prevents other berries from withering

Comment (EatPant2nd) [reply src/scripts/farming/Farming.ts]: That's perfect, thanks!

**Review by CypherX — APPROVED** [review]

**Pull request merged**